01/ht/gendos379/03/script.js

```diff
@@ -0,0 +1,78 @@
+window.onload = function() {
+    var selectIds = {'rowsNumber': 100, 'colNumber': 20}
+    for (i = 0; i < Object.keys(selectIds).length; i++) {
+        var ddl = document.getElementById(Object.keys(selectIds)[i]);
+        for (var j = 1; j <= selectIds[Object.keys(selectIds)[i]]; j++) {
+            var option = new Option;
+            option.text = j;
+            option.value = j;
+            ddl.options[j] = option;
+        }
+    }
+};
+
+var createTableButton = document.getElementById('create-table');

```

Comment:
I've made this incomplete version in a hurry, so please forgive me for this ugly code)